theano/sandbox/blocksparse.py

```diff
@@ -56,7 +56,7 @@ def make_node(self, o, W, h, inputIdx, outputIdx):
         outputIdx = theano.tensor.as_tensor_variable(outputIdx)
 
         if o.ndim != 3:
-            raise TypeError('The output o must be a 3D tensor')
+            raise TypeError('The output o must be a 2D tensor')
```

Comment:
Woops, I wrote that while I was trying to figure out what is o actually and forgot to put it back to 3. Thanks for pointing it out!